### ISSUE TITLE
Question : Routing

### ISSUE THREAD

**Issue by privatesupport** [MatsuriDayo/NekoBoxForAndroid#110]
Hi
I downloaded a "iran-geosite.db" file from this repo : https://github.com/bootmortis/iran-hosted-domains/releases
I added this file to nekobox : https://imgur.com/a/Z6zKfWv
Then I create a rule with block outbound and domain value is in this picture : https://imgur.com/a/qDE55pg
I want to test if it blocks all domains in that geosite.db file or not... bot It not blocks.
Can anyone help me? Is my config has problem... my syntax or maybe geosite file is wrong?

**Comment by EndermanCo** [comment]
1. That file is not needed.
2. You must create 2 rules with these conditions:
First rule:
name=domain law for Iran
domain=domain:ir
network=tcp and udp
outbound=bypass
The second rule:
name=IP law for Iran
ip=geoip:ir
network=tcp and udp
outbound=bypass

**Comment by privatesupport** [comment]
Yes I know that... but I want to know why extra db files do not work?

**Comment by privatesupport** [comment]
I have my custom domain list dat (v2ray,xray) and db (sing-box) files. so in sing-box i can use them but you mean in nekobox it is not possible?
